draft-linus-trans-gossip-ct.md

```diff
@@ -306,7 +308,7 @@ STHs from them.
 STH Pollination is carried out by sending STHs to HTTPS servers
 supporting the protocol, and retrieving new STHs. In the case of
 HTTPS clients, STHs are sent
-in an already established TLS session. This makes it hard for an
+in an already established TLS session [sounds like you're assuming this is done using keepalive, right? Are servers required to support keepalive?]. This makes it hard for an
```

Comment:
I can imagine this being a major implementation headache for highly optimised browsers.